docs/installation.rst

```diff
@@ -238,6 +226,36 @@ this may come up empty. To fix it, run the following::
 
     for i in /usr/local/lib/python2.7/site-packages/*mesos*; do ln -snf $i venv/lib/python2.7/site-packages/ ; done
 
+.. _docker-note:
+.. topic:: Installing Docker
+
+   `Docker`_ is needed for some of the tests. Follow the appopriate 
+   installation instructions for your system on their website to get started.
+
+   When running ``make test`` you may still get the following error
+
+   ::
+
+       Please set docker_registry, e.g. to quay.io/USER.
+
+   To solve, make an account with `Quay`_ and specify it like so: 
+
+   ::
+
+       docker_registry=quay.io/USER make test
+
+   where ``USER`` is your Quay username.
+
+   For convenience you may want to add this variable to your bashrc by running
+
+   ::
+
+       echo "export export_registry=quay.io/USER" >> $HOME/.bashrc 
```

Comment:
`docker_registry`, not `export_registry`

Make sure you actually test every command before you paste them into documentation. Don't write them from memory.